api/openapi-spec/README.md

```diff
@@ -57,4 +57,4 @@ For example:
 ### `x-kubernetes-patch-strategy` and `x-kubernetes-patch-merge-key`
 
 Some of the definitions may have these extensions. For more information about PatchStrategy and PatchMergeKey see
-[strategic-merge-patch] (https://github.com/kubernetes/community/blob/master/contributors/devel/strategic-merge-patch.md).
+[strategic-merge-patch](https://github.com/kubernetes/community/blob/master/contributors/devel/strategic-merge-patch.md).
```

Comment:
Can you also change this link to https://git.k8s.io/community/contributors/devel/strategic-merge-patch.md while you're at it? Thank you!

Reply:
I changed this link at 6b40582.